### DISCUSSION TITLE
Any plans for an ODBC connector?

### DISCUSSION THREAD

**Discussion by FactEngineCommunity** [polypheny/Polypheny-DB#454] [Ideas]
Our BI software predominantly uses ODBC connectors when a native .Net Framework database connector is not available.

I'm wondering if there is a roadmap for ODBC access to Polypheny?

**Comment by hennlo** [comment]
Hi @FactEngineCommunity indeed we are currently working on a [refactor](https://github.com/polypheny/Polypheny-DB/tree/proto-interface) of our Avatica Interface towards a ProtoBuf Interface implementation.

This will allow us to simplify the adapter and connector development. 
During this refactor the JDBC Driver and Python Connector will be adapted first.

However, if there is interest we can also try pushing an ODBC driver.
If this is important for you I suggest you open a Feature Request under [issues](https://github.com/polypheny/Polypheny-DB/issues/new?assignees=&labels=C-enhancement&template=feature_request.yml).

I'm guessing you are looking at PowerBI ? This is indeed interesting to add anyway, so feel free to open a feature request.
This is much appreciated.

**Reply by hennlo** [reply]
Thanks @FactEngineCommunity  for opening the feature request #455. We'll continue this dicussion and any new information there.
